core/block/import/importer.go

```diff
@@ -294,6 +297,18 @@ func (i *Import) getIDForAllObjects(ctx *session.Context, res *converter.Respons
 	return oldIDToNew, createPayloads, nil
 }
 
+func (i *Import) replaceRelationKeyWithNew(option *converter.Snapshot, oldIDToNew map[string]string) {
```

Comment:
do we need some tests here, what do you think?

Reply:
Do you mean only `replaceRelationKeyWithNew`? Because testing `getIDForAllObjects` makes sense with real object store

Reply:
Added test